quimb/evo.py

```diff
@@ -459,6 +462,25 @@ def _update_to_expm_ket(self, t):
         if self._step_callback is not None:
             self._step_callback(t, self._pt)
 
+    def _update_to_expm_dop(self, t):
+        """Update the (density operator) simulation to time ``t``, without 
+        explicitly computing the operator exponential itself.
+        """
+        d = self.ham.shape[0]
+        sparse = issparse(self.ham)
+        idt = eye(d, sparse=sparse)
+        evo_superop = -1.0j * (t-self.t) * ((self.ham & idt) - (idt & self.ham.T))
```

Comment:
So this step ``((self.ham & idt) - (idt & self.ham.T))`` is constant and possibly expensive, think it probably makes sense to store it somehow? Either create it in ``__init__`` or check for an attribute like ``._ham_vectrzd_expm_dop`` and only create it if it's not there at the top of this function.